.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+  "[python]": {
+    "editor.defaultFormatter": "ms-python.autopep8"
+  },
+  "python.formatting.provider": "none"
+}
```

Comment:
Hi @JoaoRoizenFontana,

Please do not include `.vscode` specific settings.

Can you delete this file?

Reply:
Of course, sorry

Reply:
Hi @misl6 ,

I created a new PR because I was having some problems when making a new commit here.

The new one is #8301 

Sorry for the inconvenience